scripts/DGSPlanner/InstrumentSetupWidget.py

```diff
@@ -271,7 +271,6 @@ def updateFigure(self):
         self.gonfig.text(1,0,-2.5,'X',zdir=None,color='black')
         self.gonfig.plot([0,0],[-3,-3],[-2,-0.5],zdir='y',color='black',linewidth=3)
         self.gonfig.text(0,-1,-2.5,'Beam',zdir=None,color='black')
-        self.gonfig.set_aspect('equal', adjustable='datalim')
```

Comment:
Was this completely removed or did the api get renamed? I'm concerned we may unintentionally be changing the default behaviour here

Reply:
It turns out this basically did completely the wrong thing in matplotlib: see matplotlib/matplotlib#13474 and it now throws a `NotImplementedError`. I guess we could catch it but it's not clear that it's actually ever going to be fixed.

Reply:
Also, if it did the wrong thing anyway, do we really want it to continue doing that?

Reply:
I'm happy with the removal, since it could change under us if upstream re-implement it